src/v2/Artsy/Router/buildServerApp.tsx

```diff
@@ -92,7 +91,7 @@ export function buildServerApp(
       })
 
       if (isRedirect(farceResults)) {
-        resolve({ redirect: farceResults.redirect, status: 302 })
+        resolve(farceResults)
```

Comment:
This object includes the status code now, which is 302 by default or whatever you pass in as the second argument.